migrations/20201216105636_van_sync_opt_out_configuration.js

```diff
@@ -0,0 +1,27 @@
+exports.up = function up(knex) {
+  return knex.schema.raw(`
+    -- Opt Out Sync Configurations
+    create table public.external_sync_opt_out_configuration (
+      id uuid not null default uuid_generate_v1mc(),
+      system_id uuid not null references public.external_system(id),
+      external_result_code_id uuid not null references public.external_result_code(id),
+      created_at timestamptz not null default now(),
+      updated_at timestamptz not null default now(),
+
+      primary key (id),
+      unique (system_id)
```

Comment:
Any reason to have a primary key other than the system_id? Doesn't seem like it's being used.